service-python/featuretoggle/src/lib/features.yml

```diff
@@ -0,0 +1,15 @@
+# Feature flags
+#
+# name:
+#  description: string describing what the flag is for
+#  enabled: boolean
+#  env?: list of enabled environments
+features:
+  foo:
+    description: an example feature flag
+    enabled: true
+    env:
+  bar:
+    description: another example feature flag
+    enabled: false
+    env:
```

Comment:
[Enabling](https://dsva.slack.com/archives/C01Q7979Z7D/p1668023719241299?thread_ts=1667862214.458039&cid=C01Q7979Z7D) `pre-commit` should autofix the last line and get rid of the red circle.